style.css

```diff
@@ -1 +1,239 @@
-/*  Add your styling here */
+@import url('https://fonts.googleapis.com/css2?family=Estonia&display=swap');
+* {
+    box-sizing: border-box;
+    
+}
+
+body {
+    max-width: 100vw;
+    font-family: 'Estonia', cursive;
+    padding-left: 1%;
+    padding-right: 1rem;
+    background: #CBB5E8;
+    color: #5F2E9E;
+    
+
+}
+
+
+.container {
+
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    
+    
+}
+/*........... header..style..start.....  */
+.hamburger {
+    width: 25px;
+    height: 15px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: space-between;
+    margin-top: 3%;
+}
+.hamburger-line {
+    width:25px;
+    background: black;
+    height: 2px;
+    
+}
+header {
+    grid-column: 1/-1;
+    background: #E8B5B8;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+}
+.logo {
+    width: 25%;
+    height: auto;
+
+}
+.logo-img {
+    height: 120px;
+    width: 120px;
+}
+.header-side-contents {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-end;
+    padding-right: 2rem;
+    text-align: right;
+    width: 50%;
+}
+/* .................header style finish .............. */
+
+
+
+/* ........navigation style start........  */
+nav {
+    grid-column: 1/-1;
+    display: none;
+}
+
+nav>ul {
+    
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+
+    
+}
+
+li {
+    list-style: none;
+    font-style: normal;
+    margin: 2%;
+    background: #E8B5B8;
+    padding: 1rem;
+}
+a {
+    text-decoration: none;
+    font-size: 1.25rem;
+    font-family: 'Courier New', Courier, monospace;
+
+}
+/* .........navigation finish......................  */
+
+
+
+/* .......welcome/intro style start...... ............ */
+.intro-texts {
+    grid-column: 1/-1;
+    padding: 0 3%;
+    letter-spacing: 1px;
+    font-size: 1.5rem;
+
+    
+
+}
+/* .......welcome/intro style finish...... ............ */
+
+
+/* ..........main/big image div style start..........  */
+.big-image {
+    height: 300px; 
+    width: 100vw;
+    background-image: url("../images/cake2.jpg");
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
```

Comment:
Do you think that this should be a background? or an image tag? What is it the background to? If it was an image tag we could add an alt tag for users who can't see the images, which is really important